keyboards/handwired/ms_sculpt_mobile/keymaps/milestogo/keymap.c

```diff
@@ -1,20 +1,28 @@
-#include "babblePaste.h"
 #include QMK_KEYBOARD_H
 
+#ifdef BOARD_GENERIC_STM32_F103
+#include "8x18_arm.h"
+#endif
+
+#include "virtser.h"
+
+#include <print.h>
 #define _QWR 0
-#define _CDH 2
-#define _SYM 3
-#define _MOV 4
+#define _CDH 1
+#define _SYM 2
+#define _MOV 3
 #define _TRAN 5
```

Comment:
You might want to use an enum here, as it will automatically number the layers, and prevent empty layers being compiled in due to gaps in the numbering (eg. this jump from 3 to 5).